x-pack/plugins/lists/common/schemas/response/exception_list_schema.mock.ts

```diff
@@ -42,6 +47,15 @@ export const getExceptionListSchemaMock = (): ExceptionListSchema => ({
   version: VERSION,
 });
 
+export const getTrustedAppsListSchemaMock = (): ExceptionListSchema => {
```

Comment:
@madirey FYI
Created this so that the client mock (below) can include it. I debated whether I should have created a sub-class of the Client under trusted apps and felt it belonged here more than in our feature code